### PR TITLE
feat(chains): add crossbell chain

### PR DESCRIPTION
**Pull request by Songkeys**
## Description

This adds [crossbell.io](https://crossbell.io) chain support.

image

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address: songkeys.eth

### CODE REVIEW

File: packages/chains/src/crossbell.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      http: ['https://rpc.crossbell.io'],
+    },
+  },
+  blockExplorers: {
+    default: { name: 'CrossScan', url: 'https://scan.crossbell.io' },
+  },
+  contracts: {
+    multicall3: {
+      address: '0xBB9759009cDaC82774EfC84D94cD9F7440f75Fcf',
+      blockCreated: 23499787,
```
Comment (tmm) [inline packages/chains/src/crossbell.ts:23]:
```suggestion
      blockCreated: 23_499_787,
```

File: packages/chains/src/crossbell.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { Chain } from './types'
+
+export const crossbell: Chain = {
+  id: 3737,
```
Comment (tmm) [inline packages/chains/src/crossbell.ts:4]:
```suggestion
  id: 3_737,
```

**Review by tmm — CHANGES_REQUESTED** [review]
Please add a patch [changeset](https://github.com/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md#versioning), then looks good to go.

**Comment by Songkeys** [comment]
Sorry I missed the review. I was on vacation. Thank you for the merge!

**Pull request merged**